drivers/misc/bh_fwtable/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2024 Tenstorrent AI ULC
+# SPDX-License-Identifier: Apache-2.0
+
+config BH_FWTABLE
+	bool "Tenstorrent Blackhole Firmware Table"
+	default y
+	depends on DT_HAS_TENSTORRENT_BH_FWTABLE_ENABLED
+	depends on TT_BOOT_FS
```

Comment:
We might be able to use `select TT_BOOT_FS` here, which would be a bit better, since people using the fwtable don't particularly need to care that there is a "filesystem" used to store the tables.